packages/fetchai/connections/http_server/connection.py

```diff
@@ -375,6 +371,7 @@ def __init__(
         super().__init__(address=address, connection_id=connection_id)
         self.host = host
         self.port = port
+        self.target_skill_id = target_skill_id
```

Comment:
Introduction of the target_skill_id means the messages generated by the server are now exclusively dealt with by a single skill.